spark-publish/bin/spark-publish.js

```diff
@@ -30,8 +29,7 @@ assert(W3UP_PROOF, 'W3UP_PROOF required')
 
 const minRoundLength = Number(MIN_ROUND_LENGTH_SECONDS) * 1000
 
-const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
-const signer = ethers.Wallet.fromMnemonic(WALLET_SEED).connect(provider)
+const signer = ethers.Wallet.fromMnemonic(WALLET_SEED)
```

Comment:
where do you now connect to the provide?

Reply:
I should have documented this, it's a refactor independent of the other changes in this PR